src/test/resources/Leas_Liudmila/autoTask5.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Automation task # 5
+
+  Background: Open Page
+    Given New Adopt Page with random location is open
+
+    Scenario: Check that A pet with an specific Name can be added to [Pets in <Location>] section
+      When A random pet name is entered and Add Rescue button is clicked
+      Then The text "No rows. Try reset filters" disappears when a pet is added
```

Comment:
O good experience is to use separate Class Messages where you can store all text messages which are related to page